src/three-components/TextureUtils.js

```diff
@@ -117,22 +123,67 @@ export default class TextureUtils extends EventDispatcher {
   }
 
   /**
-   * @param {?number} size
+   * @param {?number} config.size
+   * @param {?boolean} config.pmrem
    * @return {THREE.Texture}
    */
-  generateDefaultEnvironmentMap(size) {
-    const mapSize = size || this.config.synthesizedEnvmapSize;
+  generateDefaultEnvironmentMap(config={}) {
+    const mapSize = config.size || this.config.synthesizedEnvmapSize;
```

Comment:
Is `config.size` ever actually used anywhere?

Reply:
Nope, can remove